tests/utils/misc/misc.go

```diff
@@ -21,8 +21,29 @@ import (
 	"time"
 )
 
+type vmstatus func(string) string
+
+const (
+	maxAttempt = 60
+)
+
 // SleepForSec sleep for a given number of seconds
 func SleepForSec(sec int) {
 	log.Printf("Sleep for %d seconds", sec)
```

Comment:
It depends on the caller, I have updated existing callers to invoke this not every seconds ... BTW this is indeed a require logging to know during test execution instead of assuming a deadlock or infinite loop.